app/preparation_and_battle.py

```diff
@@ -0,0 +1,23 @@
+from app.knights import Knight
```

Comment:
Renaming the module `prepartion_and_battle.py` to battle.py is preferable as it provides a clear, concise, and easily understandable representation of the module's purpose